common-pg/src/main/scala/com/socrata/pg/store/index/FullTextSearch.scala

```diff
@@ -33,6 +34,22 @@ trait FullTextSearch[CT] {
     toTsVector(cols.toSeq)
   }
 
+  def searchNumericVector(selection: OrderedMap[ColumnName, CoreExpr[_, CT]], ctx: Option[Context]): Seq[String] = {
+    val cols = selection.view.filter(x => SearchableNumericTypes.contains(x._2.typ)).map { case (columnName, expr) =>
+      val name = ctx.map(c => qualify(columnName.name, c)).getOrElse(columnName.name)
```

Comment:
Don't really need to assign a val here. Could also use the shorthand you're using on line 48 for mapping the context for consistency.

Reply:
Removed